README.MD

```diff
@@ -27,17 +37,19 @@ Then, enable the bundle on the `AppKernel` class:
 
 ```php
 // app/AppKernel.php
-
-public function registerBundles()
-{
-    return [
-        // ...
-        new Facile\MongoDbBundle\FacileMongoDbBundle(),
-    ];
+class AppKernel extends Kernel {
```

Comment:
This is not useful with Flex, where we have a recipe: https://github.com/symfony/recipes-contrib/tree/master/facile-it/mongodb-bundle/0.6

Reply:
Btw, that recipe should be updated with the explicit `authSource: '%env(MONGODB_DB)%'`, shouldn't it?

Reply:
If you think it's a sensible default, yes.
You can duplicate the recipe and add a version constraint to 1.10 if you want.